db/migrations/20240709234116_add_status_updated_at_to_deployments.rb

```diff
@@ -0,0 +1,7 @@
+Sequel.migration do
+  change do
+    alter_table(:deployments) do
+      add_column :status_updated_at, DateTime, default: Sequel::CURRENT_TIMESTAMP, null: false
```

Comment:
**question/non-blocking** I suppose we can't set an accurate value here for existing deployments since we can't determine when the latest time status was changed right? So this field will be inaccurate until another deployment status change?

Reply:
good q... maybe we should allow null and set this in Ruby?

Reply:
Are you suggesting we set a default in Ruby or set it to null until there is a status update for existing deployments? Another option here that may be slightly closer to being correct is to set it to the value of `updated_at`.

Reply:
> set it to null until there is a status update for existing deployments?

This one.  Which means old Deployments will have a null value for last_status_change.

Using `updated_at` might be a nice compromise if we want to avoid a null column...